k8s.io/configmap-nginx.yaml

```diff
@@ -21,13 +21,21 @@ data:
       #
       # Upgrade HTTP to HTTPS.
       server {
-        server_name kubernetes.io;
-        listen 80;
-        return 301 https://$server_name$request_uri;
+        server_name k8s.io;
+        listen 80 default_server;
+
+        location /_healthz {
+          add_header Content-Type text/plain;
+          return 200 'ok';
+        }
+
+        location / {
+          return 301 https://$host$request_uri;
```

Comment:
should this go directly to https://kubernetes.io to skip a hop though https://k8s.io?